src/Classes/ConfigTab.lua

```diff
@@ -144,6 +165,7 @@ local ConfigTabClass = newClass("ConfigTab", "UndoHandler", "ControlHost", "Cont
 			if varData.type == "check" then
 				control = new("CheckBoxControl", {"TOPLEFT",lastSection,"TOPLEFT"}, 234, 0, 18, varData.label, function(state)
 					self.input[varData.var] = state
+					self.configSets[self.activeConfigSetId].input[varData.var] = state
```

Comment:
Feels odd that we need to set the configSet input + `self.input` both.  Couldn't you just make them equal and change `self.input` when you change the config set?

Reply:
Working on this, will push an update soon. Input is used in a lot of places outside ConfigTab but so long as we set it in SetActiveSet like we are and we change UpdateControls to use the self.configSets, it isn't needed with real time updates